### PR TITLE
[MBL-16688][Student][Teacher] Implement render test for left side menu

### PR DESCRIPTION
**Pull request by domonkosadam**
Test plan:

refs: MBL-16688
affects: Student, Teacher
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

File: apps/student/src/androidTest/java/com/instructure/student/ui/interaction/NavigationDrawerInteractionTest.kt [hunk 1]
```diff
@@ -129,7 +130,7 @@ class NavigationDrawerInteractionTest : StudentTest() {
         val data = MockCanvas.init(
                 studentCount = 2,
                 courseCount = 1,
-                favoriteCourseCount = 1
+                favoriteCourseCount = 1,
```
Comment (kdeakinstructure) [inline apps/student/src/androidTest/java/com/instructure/student/ui/interaction/NavigationDrawerInteractionTest.kt:133]: Please remove this unnecessary comma (,).

**Review by kdeakinstructure — APPROVED** [review]
QA 👍

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Pull request merged**